app/renderer/js/pages/preference/connected-org-section.ts

```diff
@@ -1,18 +1,27 @@
 'use strict';
 
-const BaseSection = require(__dirname + '/base-section.js');
-const DomainUtil = require(__dirname + '/../../utils/domain-util.js');
-const ServerInfoForm = require(__dirname + '/server-info-form.js');
-const AddCertificate = require(__dirname + '/add-certificate.js');
-const FindAccounts = require(__dirname + '/find-accounts.js');
+import { ipcRenderer } from 'electron';
+
+import BaseSection = require('./base-section');
+import DomainUtil = require('../../utils/domain-util');
+import ServerInfoForm = require('./server-info-form');
+import AddCertificate = require('./add-certificate');
+import FindAccounts = require('./find-accounts');
 
 class ConnectedOrgSection extends BaseSection {
-	constructor(props) {
+	// TODO: TypeScript - Here props should be object type
+	props: any;
+	$serverInfoContainer: Element | null;
+	$existingServers: Element | null;
+	$newOrgButton: Element | null;
```

Comment:
Since we know this a button make this a `HTMLButtonElement` so later if we needed to do say `.click` somebody doesn't need to fiddle with this.